agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -245,6 +260,39 @@ function install_k8s_agent {
         echo -e "        enabled: true" >> $CONFIG_FILE
     fi
 
+    if [ $INSTALL_ANALYZER -eq 1 ]; then
+      # Image Analyzer config map
+      IA_CONFIG_FILE=/tmp/sysdig-image-analyzer-configmap.yaml
+
+      # If the collector was changed but the analysis manager was not this was
+      # most likely an onprem install, add the default analysis manager for that onprem
+      if [ ! -z "$COLLECTOR" ] && [ -z "$ANALYSIS_MANAGER" ]; then
+        ANALYSIS_MANAGER="https://${COLLECTOR}/internal/scanning/scanning-analysis-collector"
+        echo "* Configuring Analysis Manager endpoint to ${ANALYSIS_MANAGER}. You can also use the -am option to explicitly specify it."
+      fi
+
+      if [ ! -z "$ANALYSIS_MANAGER" ]; then
+        echo "* Setting Analysis Manager endpoint for Image Analyzer"
+        echo "  collector_endpoint: $ANALYSIS_MANAGER" >> $IA_CONFIG_FILE
+      fi
+      if [ ! -z "$DOCKER_SOCKET_PATH" ]; then
+        echo "* Setting docker socket path"
+        echo "  docker_socket_path: $DOCKER_SOCKET_PATH" >> $IA_CONFIG_FILE
+      fi
+      if [ ! -z "$CRI_SOCKET_PATH" ]; then
+        echo "* Setting CRI socket path"
+        echo "  cri_socket_path: $CRI_SOCKET_PATH" >> $IA_CONFIG_FILE
+      fi
+      if [ ! -z "$CHECK_CERT" ]; then
```

Comment:
You should be able to remove the `else` here by defaulting CHECK_CERT. I don't think that the `else` here is doing anything.